src/Microsoft.ML.Transforms/Text/WordTokenizeTransform.cs

```diff
@@ -238,162 +181,311 @@ public override void Save(ModelSaveContext ctx)
             // *** Binary format ***
             // <base>
             // for each added column
-            //   ColInfoEx
-            SaveBase(ctx);
-            Host.Assert(_exes.Length == Infos.Length);
-            for (int i = 0; i < _exes.Length; i++)
-                _exes[i].Save(ctx);
+            //   charArray: Separators
+            SaveColumns(ctx);
+            foreach (var column in _columns)
+                ctx.Writer.WriteCharArray(column.Separators);
```

Comment:
Is this changing the format of what we save? If so, then VersionInfo should be updated right?